src/httpserver.cpp

```diff
@@ -365,10 +365,15 @@ static void HTTPWorkQueueRun(WorkQueue<HTTPClosure>* queue)
 /** libevent event log callback */
 static void libevent_log_cb(int severity, const char *msg)
 {
+#ifdef EVENT_LOG_WARN
     if (severity >= EVENT_LOG_WARN) // Log warn messages and higher without debug category
         LogPrintf("libevent: %s\n", msg);
     else
         LogPrint("libevent", "libevent: %s\n", msg);
+#else
+    // EVENT_LOG_WARN was added in 2.0.19; so in some older systems libevent warnings won't be debug-only.
+    LogPrint("libevent: %s\n", msg);
```

Comment:
Gah. What it doesn't to is it doesn't fail to compile; so much for type safety. Good spotting; thats what I get from changing from the always debug to always no debug at the last second. :-/